docs/content/Modpacks/Materials-and-Elements/01-Material-Creation.md

```diff
@@ -93,49 +60,115 @@ You can change the properties of the material by adding any combination of the f
+            .color(0x7D2DDB)
+    })
+    ```
 
-## Creating a Dust
+=== "Java"
+    ```java title="Dust.java"
+    public static Material MYSTERIOUS_DUST;
+    public static void register() {
+        MYSTERIOUS_DUST = new Material.Builder(
+            your_mod_id.id('mysterious_dust'))
+            .dust() // The haverst level and burn time can be specified in the brackets. Example: `.dust(2, 4000)`
+            .color(0x7D2DDB)
+            .buildAndRegister();
+    }
+    ```
+
+## Creating a Gem
+
+=== "JavaScript"
+    ```js title="gem.js"
+    GTCEuStartupEvents.registry('gtceu:material', event => {
+        event.create('purple_coal')
+            .gem(2, 4000) 
+            .element(GTElements.C) 
+            .ore(2, 3) 
+            .color(0x7D2DDB).iconSet(GTMaterialIconSet.LIGNITE)
 
-```js title="dust.js"
-GTCEuStartupEvents.registry('gtceu:material', event => {
-    event.create('mysterious_dust')
-        .dust()
-        .cableProperties(GTValues.V[GTValues.LV], 69, 0, true) // (1)
-})
-```
+    })
+    ```
 
-1. Voltage, Amperage, EU loss, Is Superconductor.
+=== "Java"
+    ```java title="Gem.java"
+    public static Material PURPLE_COAL;
+    public static void register() {
+        PURPLE_COAL = new Material.Builder(
+            your_mod_id.id('purple_coal'))
+            .gem(2, 4000)
+            .element(GTElements.C)
+            .ore(2, 3) 
+            .color(0x7D2DDB).iconSet(GTMaterialIconSet.LIGNITE)
+            .buildAndRegister();
+        }
+    ```
 
 ## Creating a Fluid
 
-```js title="fluid.js"
-GTCEuStartupEvents.registry('gtceu:material', event => {
-    event.create('mysterious_ooze')
-        .fluid()
-        .color(0x500bbf)
-        .fluidTemp(69420) 
-})
-```
+=== "JavaScript"
+    ```js title="fluid.js"
+    GTCEuStartupEvents.registry('gtceu:material', event => {
+        event.create('mysterious_ooze')
+          .fluid() // Or .liquid(Int Temperature)
+          .color(0x500bbf)
+    })
+    ```
+
+=== "Java"
+    ```java title="Fluid.java"
+    public static Material MYSTERIOUS_OOZE;
+    public static void register() {
+        MYSTERIOUS_OOZE = new Material.Builder(
+            your_mod_id.id('mysterious_ooze'))
+            .fluid() // Or .liquid(Int Temperature)
+            .color(0x500bbf)
+            .buildAndRegister();
+        }
+    ```
+
+
+!!! tip "Further Material Information"
+    For more information on more fine grained material control, check out the pages below!
+
+For a full list of the flags, check out the [material flags page](./Material-Flags.md).
+
+For a full list of material properties, check out the [material properties page](./Material-Properties.md).
+
+For an explanation of tools, check out the [tool creation page](./Tool_Properties.md).
+
+For an explanation of custom icon sets and a list of existing ones, check out the [icon set page](./Material-Icon-Sets.md).
```

Comment:
Capitalize the words in the clickable part of the links here so they match the page names as they are in the sidebar